frontend/pages/logout/index.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react"
+import { Layout } from "../../shared-components"
+import { Logo as NPDCLogo } from "../../shared-components"
+import styles from "../../compositions/dashboard-header/dashboard-header.module.css"
+
+import { LogoSizes } from "../../models"
+import { useAuth } from "../../helpers"
+
+export default function Logout() {
```

Comment:
Please wrap this in requireAuth, which will redirect to /login after logout is successful. Check out pages/search/index.tsx